oas_docs/README.md

```diff
@@ -1,42 +1,47 @@
 # Kibana API reference documentation
 
-Documentation about our OpenAPI bundling workflow and configuration. See Kibana's hosted [stateful](https://www.elastic.co/docs/api/doc/kibana) and [serverless](https://www.elastic.co/docs/api/doc/serverless) docs.
+Documentation about our OpenAPI bundling workflow and configuration.
+The finalized OpenAPI documents are published to [Kibana API](https://www.elastic.co/docs/api/doc/kibana) and [Kibana Serverless API](https://www.elastic.co/docs/api/doc/serverless) documentation.
 
 ## Workflow
 
 The final goal of this workflow is to produce an OpenAPI bundle containing all Kibana's public APIs.
 
 ### Step 0
 
-OAS from Kibana's APIs are continuously extracted and captured in `bundle.json` and `bundle.serverless.json` as fully formed OAS documentation. See `node scripts/capture_oas_snapshot --help` for more info.
+Kibana OpenAPI specifications are extracted from the code by `@kbn/capture-oas-snapshot-cli`.
+In particular, it captures information about HTTP APIs that use the following components:
 
-These bundles form the basis of our OpenAPI bundles to which we append and layer extra information before publishing.
+1. The Kibana Core `router` or `router.versioned` methods for defining HTTP APIs provided via the `core.http` service to all plugins
+1. Either `@kbn/config-schema` or `@kbn/zod` request and response schemas.
 
-### Step 1
-
-Append pre-existing bundles not extracted from code using [`kbn-openapi-bundler`](../src/platform/packages/shared/kbn-openapi-bundler/README.md) to produce the final resulting bundles.
-
-To add more files into the final bundle, edit the appropriate `oas_docs/scripts/merge*.js` files.
+For more information about generating this snapshot, run `node scripts/capture_oas_snapshot --help`.
 
-### Step 2
+Per [capture_oas_snapshot.sh](https://github.com/elastic/kibana/blob/main/.buildkite/scripts/steps/checks/capture_oas_snapshot.sh), OpenAPI snapshots are captured automatically for a specific list of endpoints.
```

Comment:
We should clarify what "specific" means here: all public endpoints, only a predetermined list of endpoints or something else. In other words, what does it take for an endpoint to make the cut to be included in the list?